students/philip_behrend/lesson03/mailroom.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Sep  9 22:18:06 2019
+
+@author: Philip Behrend
+"""
+import numpy as np
+
+class Donors:
+    def __init__(self,donor,amt_list):
+        self.donor = donor
+        self.amt_list = amt_list
+
+donor_list = Donors(['Marge','Harold','Henry','Myrtle','Mitchell'],
+                    [[50,40],[100,1000,10000],
+                     [2],[5,0.5,.05],[3,6,9]])
```

Comment:
fancy, but this lesson should utilize list for the donor data structure 


a common data structure for this lesson is :
```
donors = [("bob", [100])]
```